src/plugins/intel_cpu/src/cpu_streams_calculation.cpp

```diff
@@ -520,11 +522,33 @@ std::vector<std::vector<int>> generate_stream_info(const int streams,
     return proc_type_table;
 }
 
-void get_num_streams(const int streams, const std::shared_ptr<ngraph::Function>& ngraphFunc, Config& config) {
+void get_num_streams(const int streams,
+                     const std::shared_ptr<ngraph::Function>& ngraphFunc,
+                     Config& config,
+                     const std::shared_ptr<InferenceEngine::IInferencePlugin>& plugin) {
     InferenceEngine::IStreamsExecutor::Config& executor_config = config.streamExecutorConfig;
     std::vector<std::vector<int>> proc_type_table = get_proc_type_table();
 
-    generate_stream_info(streams, ngraphFunc, config, proc_type_table);
+    InferenceEngine::IStreamsExecutor::Config streamsConfig;
```

Comment:
Could we use MT2.0 interfaces?
ov::threading::IStreamsExecutor::Config

Reply:
updated